promptulate/tools/file/toolkit.py

```diff
@@ -0,0 +1,65 @@
+from typing import List, Optional
+import os
+
+from promptulate.tools.file.tools import (
+    WriteFileTool,
+    AppendFileTool,
+    ReadFileTool,
+    DeleteFileTool,
+    ListDirectoryTool,
+    CopyFileTool,
+    MoveFileTool
+)
+from promptulate.tools.base import BaseToolKit, Tool
+
+DefaultTools: List[Tool] = [
+    WriteFileTool,
+    AppendFileTool,
+    ReadFileTool,
+    DeleteFileTool,
+    ListDirectoryTool,
+    CopyFileTool,
+    MoveFileTool
+]
+
+DefaultTools_str = {
+    "WriteFileTool":WriteFileTool,
+    "AppendFileTool":AppendFileTool,
+    "ReadFileTool":ReadFileTool,
+    "DeleteFileTool":DeleteFileTool,
+    "ListDirectoryTool":ListDirectoryTool,
+    "CopyFileTool":CopyFileTool,
+    "MoveFileTool":MoveFileTool,
+}
+
+
+class FileToolKit(BaseToolKit):
```

Comment:
Maybe you can use a simpler definition like python file operation.

```python
file_tools = FileToolKit(root_dir="./xxx",modes=["read", "write", "append'])
agent = ToolAgent(tools=file_tools)
```

Reply:
use simpler definition ,like 
```python
file_tools = FileToolKit(root_dir="./xxx",modes="rwa")
agent = ToolAgent(tools=file_tools)
```